tests/ui/asm/x86_64/issue-96797.rs

```diff
@@ -1,5 +1,5 @@
 // build-pass
```

Comment:
maybe ui-test should have a way to register custom commands that can be converted to other commands. This command is just changing compiler flags, and only really something the rustc test suite cares about